### ISSUE TITLE
removes "connection" if undo to far

### ISSUE THREAD

**Issue by malkomalko** [magnars/multifiles.el#1]
if you undo in the _multifile_ buffer far enough, it will remove the duplicated item, then on redo it will add the duplicated item text back but it will no longer be mirrored and it doesn't have the strip xcross the left side telling you as such.

btw, this idea is brilliant and I would love to help and see this get some attention.

**Comment by magnars** [comment]
Yes, this package is certainly a work in progress. I think it would be doable to remove the undo-entry for inserting. My time available for Emacs-hackery is unfortunately reduced in 2013, but I'm hoping to find some time to whip this into at least a stable state. 

Thanks for the report!

**Comment by malkomalko** [comment]
I'm gonna try my hand at digging in to this package to make some updates.  the thought of being able to possibly save these working areas in the future blows my mind.

**Comment by magnars** [comment]
Very cool. I've been thinking of how that could be done in a robust way, too.
